### DISCUSSION TITLE
Package Bundles help

### DISCUSSION THREAD

**Discussion by youcrew** [Devolutions/UniGetUI#2612] [Q&A] [answered]
When I click 'New Bundle' Nothing happens. When I search for a package to add to try and start a new bundle the search does nothing.  I have over the years is have a powershell scripts for my app installs. I would love to have a bundle made to better mange installs on my computer, but I am stuck here. I have no doubt its something simple i am missing and with have a Homer Simpson moment then it is point out. Thank you in advance for reading this and any help offered.

**Comment by marticliment** [comment, marked as answer]
To add packages to a bundle, you need to add the packages from the "Discover Packages" page, select them -> Add selection to bundle

**Reply by youcrew** [reply]
Dang it, I knew i was being blind, I should have just gone to bed and looked at it with fresh eyes. Thank you for answering a silly question cordially!

**Reply by marticliment** [reply]
No, don't worry, the package bundles page needs better explaining, it is not 100% intuitive. I will try to fix it

**Comment by 42sol-eu** [comment]
I can not get it to work either.
The bundle UI stays empty on my win11 installation of UniGetUI

**Comment by TheGrChuva** [comment]
- [x] [](url)### 

>
